src/common/pricing/coingecko.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios'
+
+const getPrice = async (coingeckoId) => {
+	const url = `https://api.coingecko.com/api/v3/simple/price?vs_currencies=usd&ids=${coingeckoId}`
```

Comment:
this is for kovan or just in case can't get data from uniswap subgraph

Reply:
ok